libs/mailroom/ngx/src/lib/modules/list/pages/detail/detail.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="to-inbox" [routerLink]="['../..']">
+  <span style="padding-left: 0px" class="material-symbols-outlined" title="Back to inbox">arrow_back</span>
+  <p>Back to inbox</p>
+</div>
+
+<div class="container">
+  <div class="info-container">
+    <p>{{ ($email | async)?.from }}</p>
+    <span class="material-symbols-outlined">arrow_right_alt</span>
+    <p>{{ ($email | async)?.to }}</p>
+    <span id="spanner"></span>
+    <span id="delete" class="material-symbols-outlined" title="Delete email" (click)="deleteEmail()">delete</span>
+  </div>
+  <div>
+    <h2>{{ ($email | async)?.subject }}</h2>
+    <p>{{ ($email | async)?.text }}</p>
+  </div>
+  <ng-container *ngIf="($email | async)?.attachments?.length > 0">
```

Comment:
Linting error. 

`Object is possibly 'undefined'`

Are you seeing it?

Reply:
Yeah that was the question I had a week or so ago and you said to use nested `ng-container`s and then said "That's a rule that we can probably disable, honestly. It's very annoying. " Nesting `ng-container` didn't seem to do much for it.

Reply:
You usually have to add `ng-container` with an `ngIf` and alias the result.


``` html
 <ng-container *ngIf="($email | async) as email>">
   <ng-container *ngIf="email.attachments.length > 0">
     // This assumes email.attachments is not an optional property, otherwise you'd get the same error and would have to another level of scoped aliasing.
  </ng-container>
</ng-container>

Reply:
I mention the error because I only saw it while having the file open in my editor. It wasn't actually caught by the linter, so we can not worry about this one given the fact that we discussed modifying the linting rule for this behavior.

Reply:
On this one, since I just learned something new let's go with it so we can get rid of the error on this PR:

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#definite-assignment-assertions

Reply:
Still needs to be addressed

Reply:
Fixed.